src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,20 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        calibrateRobot(robot, toX < robot.getX() ? Direction.LEFT : Direction.RIGHT);
```

Comment:
toX < robot.getX() ? Direction.LEFT : Direction.RIGHT it better to hide this logic into method